src/bindgen/ir/constant.rs

```diff
@@ -321,8 +321,8 @@ impl Literal {
                     syn::BinOp::BitXorEq(..) => "^=",
                     syn::BinOp::BitAndEq(..) => "&=",
                     syn::BinOp::BitOrEq(..) => "|=",
-                    syn::BinOp::ShlEq(..) => ">>=",
-                    syn::BinOp::ShrEq(..) => "<<=",
+                    syn::BinOp::ShlEq(..) => "<<=",
+                    syn::BinOp::ShrEq(..) => ">>=",
```

Comment:
Looks good but would there be any chance to add a test for this?

Reply:
Upon looking at this again, I don't think it is possible to use `<<=` in constants at all, since the left hand side must be a variable. So probably these cases could also be an `unreachable!()` instead.

Reply:
Let's merge this since it's trivial, it's not a big deal.